`deprecated` module not found

### Describe your environment

OS: Windows
Python version: 3.9.7
SDK version: 1.25.0
API version: 1.25.0


### What happened?

```
Lib\site-packages\opentelemetry\trace\__init__.py:84: in <module>
    from deprecated import deprecated
E   ModuleNotFoundError: No module named 'deprecated'
```

### Steps to Reproduce

- create a new project
- setup env var
- run `poetry add opentelemetry-api`

### Expected Result

`deprecated` and `importlib-metadata` should show up in poetry.lock file as dependencies

### Actual Result

no dependencies shown for `opentelemetry-api`

### Additional context

_No response_

### Would you like to implement a fix?

Yes

--- a/opentelemetry-semantic-conventions/pyproject.toml
+++ b/opentelemetry-semantic-conventions/pyproject.toml
@@ -28,6 +28,7 @@ classifiers = [
 
 dependencies = [
   "opentelemetry-api == 1.26.0.dev",
+  "Deprecated >= 1.2.6",
 ]
 
 [project.urls]

